### DISCUSSION TITLE
Document id cannot include "/".

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [facebook/docusaurus#3183] [General]
Hi all. I got error `Document id cannot include "/".` when i try used available ids 
```
Error: Bad sidebars file. The document id 'computation-priority' was used in the sidebar, but no document with this id could be found.
Available document ids=
- advanced-guide/computation-priority
```
I try set in .md file id `advanced-guide/computation-priority` and got  `Document id cannot include "/".` 

So, how to fix it?

**Comment by slorber** [comment]
Hi

This is confusing and something I want to change, but the front matter Id
is not the "final Id" of a doc.

The id to use in sidebar is dirName/frontMatterId

Use one of the ids suggested to you in the error message and this should
work
